### PR TITLE
feat: added experimentation client project package

### PR DESCRIPTION
**Pull request by seanwilkens-ssc**
new package for issue #370
(additional PRs will be created for other packages and to update documentation)

**`solutions/experimentation/client-project`**
- includes / deprecates packages:
    - solutions/project/project-experimentation
- depends on:
    - solutions/experimentation/client-landing-zone

### CODE REVIEW

File: solutions/experimentation/client-project/network/subnet.yaml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  region: northamerica-northeast1
+  description: northamerica-northeast1 PAZ subnet
+  privateIpGoogleAccess: true
+  networkRef:
+    name: project-id-global-vpc1-vpc # kpt-set: ${project-id}-global-vpc1-vpc
+  logConfig:
+    aggregationInterval: INTERVAL_5_SEC
+    flowSampling: 0.5
+    metadata: INCLUDE_ALL_METADATA
+  secondaryIpRange:
```
Comment (davelanglois-ssc) [inline solutions/experimentation/client-project/network/subnet.yaml:44]: based on the latest discussion and documentation, kubernetes cluster will be deployed in their own subnet.. so we can remove the secondaryIprange section for this subnet

File: solutions/experimentation/client-project/network/subnet.yaml [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  region: northamerica-northeast2
+  description: northamerica-northeast2 PAZ subnet
+  privateIpGoogleAccess: true
+  networkRef:
+    name: project-id-global-vpc1-vpc # kpt-set: ${project-id}-global-vpc1-vpc
+  logConfig:
+    aggregationInterval: INTERVAL_5_SEC
+    flowSampling: 0.5
+    metadata: INCLUDE_ALL_METADATA
+  secondaryIpRange:
```
Comment (davelanglois-ssc) [inline solutions/experimentation/client-project/network/subnet.yaml:118]: based on the latest discussion and documentation, kubernetes cluster will be deployed in their own subnet.. so we can remove the secondaryIprange section for this subnet

File: solutions/experimentation/client-project/setters.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  # Client
+  ##########################
+  #
+  # Billing Account ID to be associated with this project
+  project-billing-id: "AAAAAA-BBBBBB-CCCCCC"
+  # GCP folder to use as parent to this project, lowercase K8S resource name
+  project-parent-folder: project-parent-folder
+  # user, group or serviceAccount with editor role at project level
+  project-editor: "group:team1@example.com"
+  # Naming Convention for project-id : <client-code><environment-code>m<data-classification>-<project-owner>-<user defined string>
```
Comment (davelanglois-ssc) [inline solutions/experimentation/client-project/setters.yaml:47]: we could remove this line as it is the standard we adopted for all other packages

File: solutions/experimentation/client-project/setters.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  # Max 30 characters
+  project-id: xxemu-team1-projectname
+  #
+  ##########################
```
Comment (davelanglois-ssc) [inline solutions/experimentation/client-project/setters.yaml:51]: based on the latest discussion and documentation, kubernetes cluster will be deployed in their own subnet.. so we can remove these gke setters

**Review by davelanglois-ssc — CHANGES_REQUESTED** [review]

**Review by cartyc — COMMENTED** [review]
Looks good, will approve once the changes Dave requested are pushed.

**Review by cartyc — APPROVED** [review]

**Review by fmichaelobrien — APPROVED** [review]
LGTM 
Pending requested changes

File: solutions/experimentation/client-project/README.md
Comment (davelanglois-ssc) [inline solutions/experimentation/client-project/README.md:1]: you will need to regenerate the README

**Review by cartyc — APPROVED** [review]

**Pull request merged**